lib/cmds/merge_cmds/export.ts

```diff
@@ -0,0 +1,49 @@
+import type { Argv } from 'yargs'
+
+module.exports.command = 'export'
+
+module.exports.desc = 'Export diff between two environments as a migration'
+
+module.exports.builder = (yargs: Argv) => {
+  return yargs
+    .usage('Usage: contentful merge export')
+    .option('source-environment-id', {
```

Comment:
Do we also need the space id?

Reply:
Yes! I didn't add it because it gets automatically asked for the context here https://github.com/contentful/contentful-cli/blob/14beebf03bebaea4a42ce8f014842034fa514cfe/lib/utils/middlewares.js#L67. 
The context adds the `--space-id` for most of the commands. It also allows the user to set the context before by running `contentful use <space-id>`.
I could add it as an option for clarity though

Reply:
We should provide a non-blocking way of executing the command, also if no rc file is present.